features/step_definitions/user_steps.rb

```diff
@@ -24,14 +24,14 @@
   table.hashes.each do |hash|
     User.create!(hash.merge(password: 'test1234!', organization_name: 'Agency'))
     if hash[:password_updated_at]
-     user = User.find_by_email(hash[:email])
-     user.update_attributes(password_updated_at: hash[:password_updated_at].to_date)
+      user = User.find_by_email(hash[:email])
+      user.update_attributes(password_updated_at: hash[:password_updated_at].to_date)
     end
   end
 end
 
 When /^(?:I|they) click the complete registration link in the email$/ do
```

Comment:
I'm puzzled why we're changing from the "complete registration" link to the "sites" link. Don't we still want them to visit the "complete registration" page and fill out their agency?

Reply:
That complete registration just passes a token. Yes we will later change that back. But for this release we are just going to go to the sites page.